src/Nethermind/Nethermind.AuRa.Test/AuRaBlockFinalizationManagerTests.cs

```diff
@@ -54,7 +54,7 @@ public void Initialize()
 
             _validatorStore.GetValidators(Arg.Any<long?>()).Returns(new Address[] {TestItem.AddressA, TestItem.AddressB, TestItem.AddressC});
             
-            Rlp.Decoders[typeof(BlockInfo)] = new BlockInfoDecoder(true);
```

Comment:
We don't need to replace the default one now.

Reply:
done